traffic_ops/traffic_ops_golang/apitenant/tenant.go

```diff
@@ -122,7 +123,18 @@ func (ten TOTenant) Validate() error {
 
 func (tn *TOTenant) Create() (error, error, int) { return api.GenericCreate(tn) }
 
-func (ten *TOTenant) Read() ([]interface{}, error, error, int) {
+func (ten *TOTenant) Read(h map[string][]string) ([]interface{}, error, error, int) {
+	ims := h["If-Modified-Since"]
+	var modifiedSince time.Time
+	modified := false
+	found := false
+	code := http.StatusOK
+
+	if ims != nil && len(ims) != 0 {
+		if t, err := time.Parse(time.RFC1123, ims[0]); err == nil {
```

Comment:
This isn't sufficient, HTTP Dates can be several formats. There's an existing function you can  use: https://github.com/apache/trafficcontrol/blob/7f5346618daa4d98b7fd1cf9f674650fb7fa6f7f/grove/web/util.go#L185

Which should probably be moved to `lib/go-rfc`.

See https://tools.ietf.org/html/rfc7231#section-7.1.1.1

Reply:
> _"Which should probably be moved to `lib/go-rfc`"_

:+1: